scripts/pages/main-menu/main-menu.js

```diff
@@ -85,6 +87,8 @@ class MainMenu {
 		this.panels.movie = $('#MainMenuMovie');
 		this.panels.image = $('#MainMenuBackground');
 
+		this.panels.quitButtonIcon.SetImage(`file://{images}/quit.svg`);
```

Comment:
Backticks ` instead of quotes ' are unnecessary here, backticks are for when you want to include template literals.